jpype/_jarray.py

```diff
@@ -92,6 +92,18 @@ def __new__(cls, tp, dims=1):
     def of(cls, array, dtype=None):
         return _jpype.arrayFromBuffer(array, dtype)
 
+    def __class_getitem__(cls, key):
+        if key is _jpype.JClass:
+            # explicit check for JClass
+            # _toJavaClass cannot be used
+            # passing int, float, etc is not allowed
+            key = _jpype._java_lang_Class
+        if isinstance(key, (str, _jpype._java_lang_Class)):
```

Comment:
To me, the `str` based option seems like it is adding a questionable shortcut. I don't know if this is already common practice for other `__class_getitems__` in JPype?

Reply:
https://peps.python.org/pep-0484/#forward-references

This was the first documentation I found. Just trying to address some of the comments from my phone for now.